semantics.json

```diff
@@ -289,6 +289,15 @@
     "description": "These options will let you control how the task behaves.",
     "optional": true,
     "fields": [
+      {
```

Comment:
1. If you add fields to `semantics.json`, you will have to add these fields to the language files as well. The `h5p utils update-translations` command of the H5P CLI tool can help you with that.
2. Fields are commonly sorted by their relevance, so the author will have the most commonly used ones further up. Not sure what the relevance of this option is.

Reply:
1. Good to know, I will check that out
2. I put it down as low relevance